docs/commands.md

```diff
@@ -2252,9 +2253,22 @@ hal deploy [subcommands]
 #### Subcommands
  * `details`: Get details about your currently deployed Spinnaker installation.
  * `diff`: This shows what changes you have made since Spinnaker was last deployed.
+ * `rollback`: Rollback Spinnaker to the prior version on a selected environment.
```

Comment:
You're doing a rollback via red/black, so whatever was previously deployed (and is still running but disabled) is what'll become live. It doesn't matter if the BOM version is an alias, since the subcomponents have unique tags.